bin/worker.js

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+var debug             = require('debug')('github:worker');
+var base              = require('taskcluster-base');
+var Promise           = require('promise');
+var exchanges         = require('../lib/exchanges');
+var worker            = require('../lib/worker');
+var _                 = require('lodash');
+var taskcluster       = require('taskcluster-client');
+var GitHubAPI         = require('github');
+
+/** Launch worker */
+var launch = async function(profile) {
```

Comment:
Perhaps in the future this could be refactored to provide a common runtime/context for both the worker and server process so the setting up of the config, statsdrain, influx, etc are in one location for both.